downstream/modules/platform/ref-controller-system-requirements.adoc

```diff
@@ -20,66 +20,52 @@ Runs automation. Increases memory and CPU to increase capacity for running more
 h| Requirement | Required
 | *RAM* | 16 GB
 | *CPUs* | 4
+| *Local disk* | 40GB minimum with at least 20GB available under /var/lib/awx
 |===
 
 .Control nodes
 Processes events and runs cluster jobs including project updates and cleanup jobs.
 Increasing CPU and memory can help with job event processing.
 
-
+[cols="a,a",options="header"]
 |===
 h| Requirement | Required
 | *RAM* | 16 GB
 | *CPUs* | 4
+| *Local disk* a|
+* 40GB minimum with at least 20GB available under /var/lib/awx
+* Storage volume should be rated for a minimum baseline of 1500 IOPS
+* Projects are stored on control and hybrid, and for the duration of jobs, also on execution nodes. If the cluster has many large projects, consider doubling the GB in /var/lib/awx/projects, to avoid disk space errors
```

Comment:
Is the word 'also' necessary here?

Reply:
This is indeed confusing! I've rewritten as you suggested.